invenio_accounts/ext.py

```diff
@@ -80,13 +93,29 @@ def monkey_patch_flask_security():
             recoverable.encrypt_password = encrypt_password
             registerable.encrypt_password = encrypt_password
 
-    def init_app(self, app, use_celery=True, sessionstore=None,
-                 register_blueprint=True):
+    def init_app(self, app, sessionstore=None, register_blueprint=True):
         """Flask application initialization.
 
+        The following actions are executed:
+
+        #. Initialize the configuration.
+
+        #. Monkey-patch Flask-Security.
+
+        #. Create the user datastore.
+
+        #. Create the sessionstore.
+
+        #. Initialize the extension, the forms for register users and
```

Comment:
the forms to register and confirm users ?
or: to register users and send them a confirmation
(not sure about the meaning of confirm in that context)